<launch>
	<node pkg="goal_selection" type="goal_selection_node">
		<param name="odom_frame_id" value="odom">
		<param name="base_frame_id" value="base_link">
		<param name="map_frame_id" value="map">
		<param name=" auto_state_string" value="auto">
	</node>
</launch>
